users/usersRoutes.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+
+const knex = require("knex");
+const router = express.Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const knexConfig = require("../knexfile.js");
+
+const db = knex(knexConfig.development);
+
+//JWT Secret
+const secret = "shhhhh";
```

Comment:
Put secrets in environment variables so they aren't visible on github. Even though it's not as important on this project, it's a good thing to make a habit of.